### PR TITLE
Update directoryrole-post-members.md

### PR DESCRIPTION
**Pull request by abhijeetsinha**
https://github.com/microsoftgraph/microsoft-graph-docs/issues/7828

### CODE REVIEW

File: api-reference/beta/api/directoryrole-post-members.md [hunk 1]
```diff
@@ -35,29 +35,30 @@ POST /directoryRoles/{id}/members/$ref
...
 ```http
-POST https://graph.microsoft.com/beta/directoryRoles/{id}/members/$ref
+POST https://graph.microsoft.com/beta/directoryRoles/0afed502-2456-4fd4-988e-3c21924c28a7/members/$ref
 Content-type: application/json
 Content-length: 30
 
 {
-  "directoryObject": {
-  }
+    "@odata.id":"https://graph.microsoft.com/v1.0/users/0f933635-5b77-4cf4-a577-f78a5eb090a2"
```
Comment (Lauragra) [inline api-reference/beta/api/directoryrole-post-members.md:61]: Should the ID value here also use beta?

Comment (abhijeetsinha) [reply api-reference/beta/api/directoryrole-post-members.md:62]: @Lauragra - Fixed.

Comment (Lauragra) [reply api-reference/beta/api/directoryrole-post-members.md:62]: @abhijeetsinha , this is a beta topic so the request URL should use beta, not v1.0. The question was whether the JSON content should also use beta.

**Comment by abhijeetsinha** [comment]
@Lauragra - I have resolved your comment. Can you please tell me what other change is needed?

**Comment by Lauragra** [comment]
> @Lauragra - I have resolved your comment. Can you please tell me what other change is needed?

Please use /beta in the URLs for the examples instead of /v1.0, since this is a beta topic.

**Comment by abhijeetsinha** [comment]
@Lauragra - Thanks for the guidance. I have fixed it.

**Pull request merged**